### ISSUE TITLE
Close/crash when swiping to recents

### ISSUE THREAD

**Issue by EdricTruong** [Chooloo/koler#189]
## Checklist

- [x] I am using the latest version - 1.05.55 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1. Open Koler
2. Swipe to recents

### Actual behaviour

Crashes

### Expected behavior

Switch view to recent callers

### Logs

[logcat.txt](https://github.com/Chooloo/koler/files/6307754/logcat.txt)

### Device info
 - Android version/Custom ROM version: GrapheneOS RQ2A.210405.005.2021.04.05.20
 - Device model: Pixel 3a

Previous release didn't crash for me.

**Comment by mizzunet** [comment]
> ## Checklist
> * [x]  I am using the latest version - 1.05.55
> * [x]  I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
> * [x]  This issue contains only one bug. I will open one issue for every bug report I want to file.
> 
> ### Steps to reproduce the bug
> 1. Open Koler
> 2. Swipe to recent
> 
> ### Actual behaviour
> Crashes
> 
> ### Expected behavior
> Switch view to recent callers
> 
> ### Logs
> [logcat.txt](https://github.com/Chooloo/koler/files/6307754/logcat.txt)
> 
> ### Device info
> * Android version/Custom ROM version: GrapheneOS RQ2A.210405.005.2021.04.05.20
> * Device model: Pixel 3a
> 
> Previous release didn't crash for me.

Which version of Android you're using? It doesn't seem to crash by doing this on 11 here

**Comment by EdricTruong** [comment]
@mizzunet Yeahhh I feel like this is an issue unique to me somehow..
![signal-2021-04-14-033422](https://user-images.githubusercontent.com/32976242/114697275-3dafd000-9d0d-11eb-8b08-22ef2b24e1d6.png)

**Comment by roeiedri** [comment]
By the logcat it seems you have something broken with your file system, the problem is within your specific device
I would try to clean cache and data, uninstall and reinstall the app.
Open this issue if that happens again.

**Comment by EdricTruong** [comment]
@roeiedri **Interestingly,** swiping to recents in Koler worked again without me doing anything, but it crashed when I scrolled to the bottom of recents. Tried clearing cache and data, uninstall and reinstall but it didn't change anything.

Eventually, I realized the other thing that changed was that I had made/received more calls with Koler, so my recents changed, _meaning that my old recents were pushed down_. So I figured, what if displaying a certain recent call (now lower in the list) caused this issue? I tried viewing recents in the stock app and it was fine.

I cleared my call history and went back to Koler's recents. No crashing. I guess it's a bandage solution until or if the issue comes up again. If I get any more info, I'll post it here.
